nncf/common/pruning/utils.py

```diff
@@ -238,8 +237,6 @@ def count_flops_and_weights(graph: NNCFGraph,
     Counts the number weights and FLOPs in the model for convolution and fully connected layers.
 
     :param graph: NNCFGraph.
-    :param input_shapes: Dictionary of input dimension shapes for convolutions and
-        fully connected layers. E.g {node_name: (height, width)}
     :param output_shapes: Dictionary of output dimension shapes for convolutions and
         fully connected layers. E.g {node_name: (height, width)}
```

Comment:
If `count_flops_and_weights_per_node` not need output_shapes, this function also did not use `output_shapes`.